### ISSUE TITLE
Submission notes are not associated with specific input fields

### ISSUE THREAD

**Issue by cstarcher** [TexasDigitalLibrary/Vireo#1705]
Screenreader users will not be able to associate the information panels on the right to their relevant form fields (missing aria-describedby attributes)

**Comment by cstarcher** [comment]
Note fields are not currently associated with input fields by design. This would require note fields to be assigned to input fields upon creation.
